quantum/process_keycode/process_autocorrect.c

```diff
@@ -220,12 +215,10 @@ bool process_autocorrect(uint16_t keycode, keyrecord_t *record) {
 
         if (code & 128) { // A typo was found! Apply autocorrect.
             const uint8_t backspaces = (code & 63) + !record->event.pressed;
-            if (apply_autocorrect(backspaces, (char const *)(autocorrect_data + state + 1))) {
```

Comment:
Was the `apply_autocorrect()` code here reverted accidentally?

Reply:
One too many undo commands, whoops.   Should be fixed (along with lint issues)